nerdlandbot/commands/reminder.py

```diff
@@ -0,0 +1,97 @@
+import typing
+import asyncio
+import discord
+from discord.ext import commands
+import datetime
+
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.constants import *
```

Comment:
Preferably don't use star imports (I removed another from the notify command in PR #175 )